src/prefect/runner/storage.py

```diff
@@ -276,6 +279,137 @@ def to_pull_step(self) -> Dict:
         return pull_step
 
 
+class RemoteStorage:
+    def __init__(
+        self,
+        url: str,
+        pull_interval: Optional[int] = 60,
+        **settings: Any,
+    ):
+        self._url = url
+        self._settings = settings
+        self._logger = get_logger("runner.storage.remote-storage")
+        self._storage_base_path = Path.cwd()
+        self._pull_interval = pull_interval
+
+    @staticmethod
+    def get_required_package_for_scheme(scheme: str) -> Optional[str]:
+        # attempt to discover the package name for the given scheme
+        # from fsspec's registry
+        known_implementation = fsspec.registry.get(scheme)
+        if known_implementation:
+            return known_implementation.__module__.split(".")[0]
+        # if we don't know the implementation, try to guess it for some
+        # common schemes
+        elif scheme == "s3":
+            return "s3fs"
+        elif scheme == "gs" or scheme == "gcs":
+            return "gcsfs"
+        elif scheme == "abfs" or scheme == "az":
+            return "adlfs"
+        else:
+            None
```

Comment:
should this be `return None`?

Reply:
Yeah, I don't want to raise if we don't find a corresponding package. Some protocols like FTP don't need an extra package.

Reply:
oh I meant should this be `return None` instead of just `None`

Reply:
Oh that's right, we write Python, not Ruby. I remember now.